src/statistics.rs

```diff
@@ -0,0 +1,495 @@
+use std::collections::HashMap;
+
+#[derive(Deserialize,Debug)]
+pub struct Statistics {
+    pub name: String,
+    #[serde(rename = "type")]
+    pub client_type: String,
+    pub ts: i64,
+    pub time: i64,
+    pub replyq: i64,
+    pub msg_cnt: i64,
+    pub msg_size: i64,
+    pub msg_max: i64,
+    pub msg_size_max: i64,
+    pub simple_cnt: i64,
+    pub brokers: HashMap<String, Broker>,
+    pub topics: HashMap<String, Topic>,
+    pub cgrp: Rebalance
+}
+
+#[derive(Deserialize,Debug)]
+pub struct Broker {
+    pub name: String,
+    pub nodeid: i32,
+    pub state: String,
+    pub stateage: i64,
+    pub outbuf_cnt: i64,
+    pub outbuf_msg_cnt: i64,
+    pub waitresp_cnt: i64,
+    pub waitresp_msg_cnt: i64,
+    pub tx: i64,
+    pub txbytes: i64,
+    pub txerrs: i64,
+    pub txretries: i64,
+    pub req_timeouts: i64,
+    pub rx: i64,
+    pub rxbytes: i64,
+    pub rxerrs: i64,
+    pub rxcorriderrs: i64,
+    pub rxpartial: i64,
+    pub zbuf_grow: i64,
+    pub buf_grow: i64,
+    pub wakeups: i64,
+    pub int_latency: Window,
+    pub rtt: Window,
+    pub throttle: Window,
+    pub toppars: HashMap<String, TopicPartition>
+}
+
+#[derive(Deserialize,Debug)]
+pub struct Window {
+    pub min: i64,
+    pub max: i64,
+    pub avg: i64,
+    pub sum: i64,
+    pub cnt: i64
+}
+
+#[derive(Deserialize,Debug)]
+pub struct TopicPartition {
+    pub topic: String,
+    pub partition: i32
+}
+
+#[derive(Deserialize,Debug)]
+pub struct Topic {
+    pub topic: String,
+    pub metadata_age: i64,
+    pub partitions: HashMap<i32, Partition>
+}
+
+#[derive(Deserialize,Debug)]
+pub struct Partition {
+    pub partition: i32,
+    pub leader: i32,
+    pub desired: bool,
+    pub unknown: bool,
+    pub msgq_cnt: i64,
+    pub msgq_bytes: i64,
+    pub xmit_msgq_cnt: i64,
+    pub xmit_msgq_bytes: i64,
+    pub fetchq_cnt: i64,
+    pub fetchq_size: i64,
+    pub fetch_state: String,
+    pub query_offset: i64,
+    pub next_offset: i64,
+    pub app_offset: i64,
+    pub stored_offset: i64,
+    pub committed_offset: i64,
+    pub eof_offset: i64,
+    pub lo_offset: i64,
+    pub hi_offset: i64,
+    pub consumer_lag: i64,
+    pub txmsgs: i64,
+    pub txbytes: i64,
+    pub msgs: i64,
+    pub rx_ver_drops: i64
+}
+
+#[derive(Deserialize,Debug)]
+pub struct Rebalance {
+    pub rebalance_age: i64,
```

Comment:
The same actually apply to many other data structures. Let's leave it as is, it's unlikely to be a problem in practice.